pkg/route/registry/route/strategy.go

```diff
@@ -214,6 +214,26 @@ func (s routeStrategy) validateHostUpdate(ctx apirequest.Context, route, older *
 		if hostChanged {
 			return kvalidation.ValidateImmutableField(route.Spec.Host, older.Spec.Host, field.NewPath("spec", "host"))
 		}
+	}
+	tlsRes, err := s.sarClient.Create(
+		authorizationutil.AddUserToSAR(
```

Comment:
RBAC would not let you invoke an update to a route if you did not already have this permission.  So this SAR does not make sense since it would always return true.